apps/heroku/src/inngest/functions/users/delete-user.ts

```diff
@@ -0,0 +1,48 @@
+import { eq } from 'drizzle-orm';
+import { NonRetriableError } from 'inngest';
+import { db } from '@/database/client';
+import { env } from '@/env';
+import { Organisation } from '@/database/schema';
+import { deleteUser } from '@/connectors/users';
+import { inngest } from '../../client';
+
+export const deleteHerokuUser = inngest.createFunction(
+  {
+    id: 'heroku-delete-user',
+    priority: {
+      run: '600',
+    },
+    retries: env.REMOVE_ORGANISATION_MAX_RETRY,
+  },
+  {
+    event: 'heroku/users.delete.requested',
+  },
+  async ({ event, step }) => {
+    const { id, organisationId } = event.data;
+
+    // retrieve the Heroku organisation access token and team Id
+    const [accessToken, teamId] = await step.run('get-access-token', async () => {
```

Comment:
Let's make the step title clearer
```suggestion
    const organisation = await step.run('get-organisation', async () => {
```